### PR TITLE
Item category

### PR DESCRIPTION
**Pull request by paulgrever**

### CODE REVIEW

File: app/views/admin/items/edit.html.erb [hunk 1]
```diff
@@ -1,3 +1,11 @@
 <h1>Edit <%= @item.title %> </h1>
 <%= render 'form' %>
+
+
+<h4>Add Categories to Item</h4>
+
+
```
Comment (jgujgu) [inline app/views/admin/items/edit.html.erb]: Too many new lines here.

File: app/helpers/categories_helper.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 module CategoriesHelper
-end
+  def categoy_all
```
Comment (jgujgu) [inline app/helpers/categories_helper.rb]: category misspelled

File: app/views/admin/categories/_update_categories.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="row">
+  <div class="col-lg-6">
+    <div class="input-group">
+      <span class="input-group-addon">
+        <% categoy_all.each do |category| %>
```
Comment (jgujgu) [inline app/views/admin/categories/_update_categories.html.erb]: category misspelled

File: app/views/admin/items/edit.html.erb (hunk 1 above)
Comment (jgu2160) [inline app/views/admin/items/edit.html.erb]: cateogory_ids --> category_ids

File: app/views/admin/categories/_update_categories.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+          <%= check_box_tag 'cateogory_ids[]', category.id, @item.categories.include?(category) ? true : false-%>
```
Comment (jgujgu) [inline app/views/admin/categories/_update_categories.html.erb]: cateogory_ids --> category_ids

File: app/views/admin/items/index.html.erb [hunk 5]
```diff
@@ -1,7 +1,11 @@
-<h1>Welcome to the Road Kill Cafe</h1>
-<ul>
+<h1>All Menu Item List:</h1>
+<div class="row">
   <% @items.each do |item| %>
-    <li> <%= link_to item.title, admin_item_path(item) %> </li>
+    <div class="box col-sm-3">
+      <%= image_tag item.image, size: "100" %><br>
+      <%= link_to item.title, admin_item_path(item) %><br>
+
```
Comment (jgujgu) [inline app/views/admin/items/index.html.erb]: delete blank line

**Comment by loriculberson** [comment]
Great job!

**Pull request merged**